hotload.py

```diff
@@ -46,6 +46,21 @@ def _file_changed(f):
 def _all_file_changes(filepaths):
     return {path: _file_changed(path) for path in filepaths}
 
+def _changed_modules(new_changed, last_changed):
+    list = []
+    if last_changed is None:
+        return list
+    modules = {}
+    for module in sys.modules.copy().values():
+        file = getattr(module, '__file__', None)
+        if file is not None:
+            modules[file] = module
```

Comment:
(Could this module loop be a performance issue? An alternative could be to store the modules similar to `watchfiles`)

Reply:
This far I haven't really explored possible performance hits. Polling is bad (in general).

Why?

I simply haven't yet needed hotload when working on codebases that are so big that performance _becomes_ an issue.